src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/SignedXml.cs

```diff
@@ -779,19 +782,19 @@ private static IList<string> DefaultSafeTransformMethods
             }
         }
 
-        private byte[] GetC14NDigest(HashAlgorithm hash)
+        private byte[]? GetC14NDigest(HashAlgorithm hash)
```

Comment:
Pretty sure this can't return null.

Reply:
Line `789` says:
`if (isKeyedHashAlgorithm || !_bCacheValid || !SignedInfo!.CacheValid)`
In that `if` block is the only place where `_digestedSignedInfo` is set, so if the expression is `false`, then it'll return `null`.
But I'm not familiar with how this type works, so it could be that it's impossible for this expression to be `false` if it hasn't previously been `true`.
Looking at the usages of the return value, the consumers don't expect it to be `null`.
So I've changed it.